bob-backend/src/pearl/group.rs

```diff
@@ -206,10 +224,17 @@ impl Group {
         timestamp_config: StartTimestampConfig,
     ) -> Result<(), Error> {
         let holder = self.get_actual_holder(&data, timestamp_config).await?;
-        Self::put_common(holder, key, data).await
+        let res = Self::put_common(&holder.1, key, data).await;
```

Comment:
Add `?` here. And ignore `Ok` result. Or we will completely miss errors from pearl on `put`